build.sbt

```diff
@@ -49,7 +49,7 @@ BundleKeys.systemVersion := "2"
 
 BundleKeys.endpoints := Map(
   "akka-remote" -> Endpoint("tcp"),
-  "web" -> Endpoint("http", services = Set(URI("http://conductr.lightbend.com")))
+  "web" -> Endpoint("http", 0, acls = RequestAcl(Http("^/".r))))
```

Comment:
Custom HAProxy template declaration is required - paste the following declaration in between the `{{#eachAcls}}` block:

```
  {{#ifAcl 'doc-renderer-cluster' '2' 'web'}}
frontend doc_frontend
  bind {{haproxyHost}}:80
  mode http
  acl doc_host_match hdr(host) -i conductr.lightbend.com
  use_backend doc_backend if doc_host_match

backend doc_backend
  mode http
    {{#eachBackendServer}}
  server {{serverName}} {{host}}:{{port}} maxconn 1024
    {{/eachBackendServer}}
  {{/ifAcl}}
```